ata_pipeline0/helpers/preprocessors.py

```diff
@@ -119,6 +121,7 @@ class ConvertFieldTypes(Preprocessor):
     fields_float: Set[FieldSnowplow]
     fields_datetime: Set[FieldSnowplow]
     fields_categorical: Set[FieldSnowplow]
+    fields_json: Set[FieldSnowplow]
```

Comment:
🌱 I'm assuming we won't be needing this once we've extracted what we want out of the JSON form fields (and hopefully have our partners restrict form tracking to just newsletter forms).